### DISCUSSION TITLE
Battery Consumption

### DISCUSSION THREAD

**Discussion by NeeRaj-2401** [z-huang/music#402] [Q&A]
I have realme 7 device with realme ui 3.0 android 12.
in realme ui 3.0 user can monitor battery usage of any app in settings ->battery
so, i noticed innerTune consuming more battery (mAh) as compared to Youtube Music or Vimusic as per my daily usage

**Comment by z-huang** [comment]
Huh, good question, but not easy to fix. I'll observe that in my phone first.
